About/About/who-are-we.md

```diff
@@ -1,12 +1,18 @@
+
+
 ## Who are we?
 
-CHAOSS is a Linux Foundation project focused on creating analytics and metrics to help define community health. Work in the CHAOSS Project community is organized in two committees and four working groups:
+CHAOSS(Community Health Open Source Analytics Software) is a Linux Foundation project creates tools to analyze software development and project health along several dimensions as well as robust guidelines and focused on creating the analytics and metrics making an open source community healthy and sustainable. 
+
+Work in the CHAOSS Project community is organized in two committees and five working groups:
```

Comment:
What did you try to improve here? 
What was the shortcoming of the existing text?